ci-operator/config/openshift/verification-tests/openshift-verification-tests-master__installation-nightly-4.16.yaml

```diff
@@ -333,9 +333,9 @@ tests:
     test:
     - chain: cucushift-installer-check-cluster-health
     workflow: cucushift-installer-rehearse-ibmcloud-ipi-private-sshkey
-- as: nutanix-ipi-zones-single-osimage-customized-reboot-nodes-f28
+- as: nutanix-ipi-zones-single-osimage-customized-reboot-fields-f28
```

Comment:
I suggest to prepare a separate install workflow to cover the invalid fields testing.